src/shared/navgen/nav.cpp

```diff
@@ -1061,9 +1061,17 @@ void NavmeshGenerator::Init(Str::StringRef mapName)
 	LoadGeometry();
 	if ( initStatus_.code != NavgenStatus::OK ) return;
 
+	cellHeight = config_.cellHeight;
```

Comment:
Let's name the one in config to `requestedCellHeight` to reduce chances for confusion

Reply:
done